Modules/Quotas/src/Quotas.Application/Identities/Queries/GetQuotasForIdentity/ListQuotasForIdentityQuery.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Backbone.BuildingBlocks.Application.Pagination;
+using MediatR;
+
+namespace Backbone.Modules.Quotas.Application.Identities.Queries.GetQuotasForIdentity;
+
+public class ListQuotasForIdentityQuery : IRequest<ListQuotasForIdentityResponse>
+{
+    public ListQuotasForIdentityQuery(PaginationFilter paginationFilter, string address)
+    {
+        PaginationFilter = paginationFilter;
+        Address = address;
+    }
+
+    public PaginationFilter PaginationFilter { get; set; }
```

Comment:
Pagination is only necessary for requests where we expect there to be a lot of response items. Which is not the case for quotas. I guess there will be at most 10.